TASK-4/CandidateReportGenerator.cs

```diff
@@ -0,0 +1,32 @@
+namespace Persons;
+
+public class CandidateReportGenerator : IReportGenerator
+{
+    public IEnumerable<Candidate> ListOfCandidates;
+
+    public CandidateReportGenerator(IEnumerable<Candidate> listOfCandidates)
+    {
+        ListOfCandidates = listOfCandidates;
+    }
+
+    public void Report()
+    {
+        const string template = "{0,36} | {1,20} | {2, -40} | {3,25} | {4, 18}";
+
+        Console.WriteLine(template,
+            "Id:", "Name Surname:", "Desired position:", "Position description:", "Desired salary:");
+        for (int i = 0; i < ListOfCandidates.Count(); i++)
+        {
+            var orderedCandidates = from selectedCandidate in ListOfCandidates
+                                    orderby selectedCandidate.DesiredPosition, selectedCandidate.DesiredSalary
+                                    select selectedCandidate;
```

Comment:
Sorting by salary isn't working

Reply:
As far as I can see, the sorting is working, have a look at the picture please:

![sorting](https://user-images.githubusercontent.com/98948073/158353586-778bfd23-1fb0-4f48-b3fc-baac7676e6e4.png)

Reply:
Yes, the sorting works correctly, ok.